lib/power_action_utils.pm

```diff
@@ -282,6 +282,41 @@ sub assert_shutdown_and_restore_system {
             select_console($vnc_console);
         }
     }
+
+=head2 assert_shutdown_with_soft_timeout
+
+  assert_shutdown_with_soft_timeout({[,timeout => $timeout] [,soft_timeout => $soft_timeout] [, bugref => $bugref] [,soft_failure_reason => $soft_failure_reason]});
```

Comment:
can you remove the leading comma for the "timeout" parameter?

Reply:
@okurz, I'm not too familiar with this syntax and just copied the approach from your 'assert_screen_with_soft_timeout' function, though the last one contains first mandatory argument $mustmatch. 

If I correctly understood, the comma means that the argument is optional. Am I right?